.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.tmx -diff
+*.tsx -diff
```

Comment:
these lines make git treat these files like binary, i.e. not try and merge them in case of conflicts